assets/src/edit-story/components/panels/shared/styles.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { css } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { themeHelpers } from '../../../../design-system/theme';
+
+export const focusStyle = css`
+  ${({ theme }) =>
+    themeHelpers.focusableOutlineCSS(
```

Comment:
Would it be an issue if this helper outputted `:focus, :focus-within {...}`, which would make these two constants the same? Would that simplify things?

Reply:
Hm, perhaps, but not sure if this could cause some issue. E.g. if there are cases where a focusable element can be inside another focusable element. Anyway, I'll leave it as it is currently, we can investigate it later in a separate PR.